src/rest-api/stats/class-endpoint-posts.php

```diff
@@ -237,6 +260,11 @@ public function get_posts( WP_REST_Request $request ) {
 
 		// Process the data.
 		$posts = array();
+		/**
```

Comment:
Let's add a blank line above this.

```suggestion

		/**
```